### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by Uthmanbello**
In this project:
1. Mobile menu was created
2. Javascript was used for the mobile hamburger
3. Linters were used to clear errors

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -1238,3 +1161,128 @@ body {
...
+@media (max-width: 768px) {
+  .header {
+    width: 332px;
+  }
+
+  .nav-icon {
+    display: block;
+  }
+
+  li a {
```
Comment (brainconnect93) [inline styles.css:1232]: - [ ] having made a great effort working on the functionality of the mobile menu when I click on the `hamburger`, the mobile menu links appear, and when I click on the `(X)`, the mobile menu links disappear and the links direct me to the necessary page in the webpage. But currently, I notice some wrong alignments of the `mobile version` of the project as it is in the Figma design. Kindly check the `mobile version` of your project on the `screen size of 375px` and make it align well with the screen

Your Design 👇 

bel-mob

Figma Design 👇 

real-mob

>N.B: I just picked the line of codes above in order to make an in-line comment, not necessarily where the error is coming from. Kindly check the styling to rectify the alignment

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @Uthmanbello , 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (Uthmanbello) [reply styles.css:1232]: @brainconnect93 Good day. First of all, congratulations on your award yesterday!! I have tried my best to get rid of the extra space but couldn't. Please, can you help me with a way out?

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark:

Hi @Uthmanbello, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

_**Kindly check an optional comment I left under the review**_ :pray: 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README, but not professional. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [OPTIONAL] I would like to recommend you to include some very important sections to your README file, such as `Key Features` and `Future Features` in order to make your README file looks professional.

**Pull request merged**